ffv1.md

```diff
@@ -288,29 +288,36 @@ For each `Plane` of each slice, a Quantization Table Set is selected from an ind
 
 - For Y `Plane`, `quant_table_set_index [ 0 ]` index is used
 - For Cb and Cr `Planes`, `quant_table_set_index [ 1 ]` index is used
-- For Alpha `Plane`, `quant_table_set_index [ (version <= 3 || chroma_planes) ? 2 : 1 ]` index is used
+- For extra `Plane`, `quant_table_set_index [ (version <= 3 || chroma_planes) ? 2 : 1 ]` index is used
 
 Background: in first implementations of FFV1 bitstream, the index for Cb and Cr `Planes` was stored even if it is not used (chroma_planes set to 0), this index is kept for version <= 3 in order to keep compatibility with FFV1 bitstreams in the wild.
 
 ## Color spaces
 
-FFV1 supports two color spaces: YCbCr and RGB. Both color spaces allow an optional Alpha `Plane` that can be used to code transparency data.
+FFV1 supports several color spaces. The count of allowed coded planes and the meaning of the extra `Plane` are determined by the selected color space.
 
 The FFV1 bitstream interleaves data in an order determined by the color space. In YCbCr for each `Plane`, each `Line` is coded from top to bottom and for each `Line`, each `Sample` is coded from left to right. In JPEG2000-RCT for each `Line` from top to bottom, each `Plane` is coded and for each `Plane`, each `Sample` is encoded from left to right.
 
 ### YCbCr
 
-In YCbCr color space, the Cb and Cr `Planes` are optional, but if used then MUST be used together. Omitting the Cb and Cr `Planes` codes the frames in grayscale without color data. An FFV1 `Frame` using YCbCr MUST use one of the following arrangements:
+This color space allows 1, 3 or 4 `Planes`.  
```

Comment:
But 2 plane are allowed, right? As in `Y, Transparency`?

Reply:
Oops, fixed.